pom.xml

```diff
@@ -1139,37 +1139,38 @@
       <dependency>
         <groupId>org.scalatest</groupId>
         <artifactId>scalatest_${scala.binary.version}</artifactId>
-        <version>3.3.0-SNAP3</version>
+        <version>3.2.13</version>
```

Comment:
This seems like a downgrade?

Reply:
Not downgrade:

1. https://search.maven.org/artifact/org.scalatest/scalatest_2.12
<img width="977" alt="image" src="https://user-images.githubusercontent.com/1475305/189393708-d31defe2-08ea-41e5-956f-21548cc72773.png">

The update time of `3.3.0-SNAP3` is 11-Dec-2020,  this is earlier than the release of 3.2.4

2. https://github.com/scalatest/scalatest/tags

There is no 3.3.x officially tag and it seems that the 3.3.x branch is still under development or has been abandoned(The latest commit was 16 Mar)

So I don't think we should use this version

Reply:
Later in time doesn't mean later version. If you're just saying this seems like a somewhat abandoned 3.3.x release, OK